nerves-hub/cli-tools.md

```diff
@@ -0,0 +1,61 @@
+---
+description: >-
+  Introducing the NervesHub command-line tools aka. the CLI.
+---
+
+# CLI tools
+
+## Installation
+
+The easiest way to install is via [Homebrew](https://brew.sh/).
+
+```
+brew install nerves-hub/tap/nh
+```
+
+The second easiest is via curl:ing a shell script.
+
+```
+curl --proto '=https' --tlsv1.2 -fsSL https://raw.githubusercontent.com/nerves-hub/nerves_hub_cli/master/install.sh | sh
+```
+
+More details and alternative installation methods are available in [the repo](https://github.com/nerves-hub/nerves_hub_cli).
+
+## Setup
+
+To set your NervesHub instance to use with the CLI, use this command, replacing the specific URL with your instance:
+
+```sh
+nh config set uri "https://manage.nervescloud.com/"
+```
+
+You need to be authorized with the NervesHub instance, you get that via:
+
+```sh
+nh user whoami
```

Comment:
you can remove this line as this only works after auth

Reply:
Oh, I took that from the nerves_hub_link instructions. I assume it would show something helpful :D